x-pack/legacy/plugins/apm/public/components/app/TransactionDetails/WaterfallWithSummmary/TraceSummary/Duration.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiToolTip, EuiText } from '@elastic/eui';
+import styled from 'styled-components';
+import { asTime } from '../../../../../utils/formatters';
+import { PercentOfTrace } from '../PercentOfTrace';
+
+interface DurationProps {
+  duration: number;
+  totalDuration?: number;
+}
+
+const Span = styled('span')`
+  white-space: nowrap;
+`;
+
+export function Duration({ duration, totalDuration }: DurationProps) {
+  totalDuration = totalDuration || duration;
+  const label = i18n.translate('xpack.apm.transactionDetails.durationLabel', {
+    defaultMessage: 'Duration'
+  });
+
+  return (
+    <Span>
+      <EuiToolTip content={label}>
+        <EuiText aria-label={label}>{asTime(duration)}</EuiText>
```

Comment:
As far as I know, you have to add the value in the aria-label too. The screenreader will drop the other text.

Reply:
Just tried it with Chrome Vox Extension and I got only the label info - not the value.

Reply:
It looks like this isn't necessary anyway because EUI automatically adds an `aria-describedby` to the tooltip (https://github.com/elastic/eui/blob/849f52aace38081328d24efa8079edf82289dc3c/src/components/tool_tip/tool_tip.tsx#L315).

Thanks for pointing that out though. I learned something.